frontend/cypress/integration/common/wizard_istio_config.ts

```diff
@@ -19,6 +19,20 @@ When('viewing the detail for {string}', (object: string) => {
   cy.get('a').contains(object).click();
 });
 
+When ('k8sgateway {string} is not created',(name:string) =>{
+  cy.exec(`kubectl delete gateways.gateway.networking.k8s.io ${name} -n bookinfo`, { failOnNonZeroExit: false });
+  ensureKialiFinishedLoading();
+});
+
+When ('gateway {string} is not created',(name:string) =>{
+  cy.exec(`kubectl delete gateway.networking.istio.io ${name} -n bookinfo`, { failOnNonZeroExit: false });
+  ensureKialiFinishedLoading();
+});
+
+When ('service {string} is not created',(name:string) =>{
```

Comment:
please adjust this Gherkin and step definition to something a bit more comprehensible

`user deletes service named {string} and resource no longer available`